pkg/sentry/server/validator/kubernetes/kubernetes.go

```diff
@@ -192,10 +187,18 @@ func (k *kubernetes) Validate(ctx context.Context, req *sentryv1pb.SignCertifica
 	// for the ID containing their namespace and service account (ns:sa). This
 	// is wrong- dapr identities are based on daprd namespace + _app ID_.
 	// Remove this allowance in v1.13.
-	if req.Namespace+":"+pod.Spec.ServiceAccountName == req.Id {
+	if pod.Namespace+":"+pod.Spec.ServiceAccountName == req.Id {
```

Comment:
Better to use fmt.Sprintf

Reply:
We don't want to use `fmt.Sprintf` here as this is a very hot path for Dapr and bare string concatenation `+` is more performant.